packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -165,14 +160,6 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
             }
         });
 
-        commands.registerCommand(VscodeCommands.SET_CONTEXT, {
-            isVisible: () => false,
```

Comment:
@akosyakov 
I guess the the command was registered for https://github.com/eclipse-theia/theia/issues/4053

How it can be handled after removing this part?
Do you mean it's registered on VS Code side?

Sorry, if I missed something...

Reply:
It is already registered by Monaco. On master you can see a warning that this registration is ignored.

Reply:
thanks!